overrides/inspec/resource_override.rb

```diff
@@ -22,6 +22,8 @@ def self.attributes
         %i[
           manual
           additional_functions
+          product_url
```

Comment:
Don't we have a way of overriding the collection_url already?

(Might've imagined that)

Reply:
It's not the collection_url, it's the base `googleapis/v1/` so that I can use a v2 API for some resources without changing the other resources (they aren't on v2)